Application/Features/SendRecurringInfoFeature/Queries/GetCurrencySymbolsListQuery.cs

```diff
@@ -11,32 +11,69 @@ public class GetCurrencySymbolsListQueryRequest : IRequest<GetCurrencySymbolsLis
 
     public class GetCurrencySymbolsListQueryHandler : IRequestHandler<GetCurrencySymbolsListQueryRequest, GetCurrencySymbolsListQueryResponse>
     {
+        private readonly HttpClient _httpClient;
+        private readonly string _apikey = "uHnC6f1zjtWiQbIyjUAcdu48L2uoom8V";
+
```

Comment:
Read static texts from configuration. You can write them in the appsettings.local file because apikey will be unique for every developer